build/cmake/CMakeModules/AddZstdCompilationFlags.cmake

```diff
@@ -20,7 +29,11 @@ function(EnableCompilerFlag _flag _C _CXX _LD)
         endif ()
     endif ()
     if (_LD)
-        CHECK_LINKER_FLAG(C ${_flag} LD_FLAG_${varname})
+        if (ZSTD_HAVE_CHECK_LINKER_FLAG)
```

Comment:
Can you just add a comment to this function that says that it doesn't add ldflags for older cmake versions? In case that we need to add other ldflags that are non-optional later.

Reply:
It makes sense. I've add a comment:

```cmake
        # We never add a linker flag with CMake < 3.18. We will
        # implement CHECK_LINKER_FLAG() like feature for CMake < 3.18
        # or require CMake >= 3.18 when we need to add a required
        # linker flag in future.
```

Does this make sense?